src/main/resources/templates/configuration.html

```diff
@@ -12,7 +12,7 @@
             <form method="post" th:action="@{/configurationSaved}" th:object="${endpointConfiguration}" class="form-horizontal">
 
                 <div class="form-group">
-                    <label for="uri" class="col-sm-2 control-label">URI</label>
+                    <label for="uri" class="col-sm-2 control-label">Host</label>
```

Comment:
Just to make sure, you are requesting changes only in the `EndpointConfiguration` class? Because going further from that class, there are ripple changes throughout the entire `dataProvider` and `runtime` codebase.